modules/core/src/test/scala/org/scalasteward/core/repoconfig/RepoConfigAlgTest.scala

```diff
@@ -278,7 +278,21 @@ class RepoConfigAlgTest extends FunSuite {
     assert(config.maybeRepoConfig.isDefined)
   }
 
-  test("config file in .config/") {
+  test("not hidden config file in .github/") {
+    val repo = Repo("test", "dot-github-config")
+    val repoDir = workspaceAlg.repoDir(repo).unsafeRunSync()
+    val rootConfigFile = repoDir / "scala-steward.conf"
+    val dotGithubConfigFile = repoDir / ".github" / "scala-steward.conf"
+    val initialState = MockState.empty.addFiles(dotGithubConfigFile -> "").unsafeRunSync()
+    val config = repoConfigAlg.readRepoConfig(repo).runA(initialState).unsafeRunSync()
+
+    assert(!fileAlg.isRegularFile(rootConfigFile).unsafeRunSync())
+    assert(fileAlg.isRegularFile(dotGithubConfigFile).unsafeRunSync())
+
+    assert(config.maybeRepoConfig.isDefined)
+  }
```

Comment:
The tests I added a basically just duplicates of the preceding test, just with the dot removed from the filename.